css-showcases/src/CssShowcases.js

```diff
@@ -1,10 +1,10 @@
-import '~/showcases/dockit-showcases.define.js';
+import '@divriots/dockit-core/showcases/dockit-showcases.define.js';
 import { getCssCustomProps } from './css-props';
 import { getZIndexHtml } from './z-index-helper';
 import { getScaleHtml } from './space-helper';
 import { getTransitionsHtml } from './transition-helper';
 import styles from './alerts.module.scss';
-import '~/clipboard/dockit-clipboard.define.js';
+import '@divriots/dockit-core/clipboard/dockit-clipboard.define.js';
```

Comment:
this is not where it's good to use bare module specifiers
the idea to use it in docs comes from the fact that docs code pieces will be presented to users, right? so that need to see the API from the external perspective
everywhere else, including in sources, in stories and so on you can use relative paths or `~/` paths

in short the rule is simple
- is the code visible to users reading documentation? then use bare module specifier
- is the code not visible: use `~/` or relative imports

Reply:
Yes, I understood the rule before - this is a "replace all" mistake.
Will fix, thanks.